seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
             case BIGINT:
-                return Bytes.toLong(cell);
+                return Long.valueOf(Bytes.toString(cell));
             case FLOAT:
             case DECIMAL:
-                return Bytes.toFloat(cell);
             case DOUBLE:
-                return Bytes.toDouble(cell);
+                return Double.valueOf(Bytes.toString(cell));
```

Comment:
**Issue**: The conversion from bytes to string and then to double is not only inefficient but also risky if the byte array does not accurately represent a double. <br> **Fix**: It is advisable to use direct conversion from bytes to double to enhance performance and reduce error likelihood. <br> **Code Suggestion**: 
 ```
 -                return Double.valueOf(Bytes.toString(cell));
 +                return Bytes.toDouble(cell);
 ```